storybook/stories/Layout/LightBox.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { LightBox } from "../../../src/components/Layout/LightBox";
+import { AppText } from "../../../src/components/Layout/AppText";
+import { Dimensions, View } from "react-native";
+
+storiesOf("Layout/LightBox", module)
+  .add("QR", () => (
+    <View>
+      <LightBox
+        width={Dimensions.get("window").width * 0.7}
+        height={Dimensions.get("window").width * 0.7}
+        label={<AppText>Scan QR Code</AppText>}
```

Comment:
Please style the `AppText` to at least allow it to be seen. Either change the color, or change the background of this story to be dark

Reply:
updated